packages/integration-tests/tests/ui/transactions.spec.ts

```diff
@@ -38,3 +44,82 @@ test("Verify deployed the own ERC20 token contract", async ({ page }) => {
 
   await expect(element).toBeVisible(config.extraTimeout);
 });
+
+//@id1682
+test(" Check on BE Transfer ETH token via Portal", async ({ page }) => {
+  transaction = await helper.readFile(Path.absolutePathToBufferFiles, Buffer.txEthWithdraw);
+  url = BlockExplorer.baseUrl + `/tx/${transaction}` + BlockExplorer.localNetwork;
+
+  await page.goto(url);
+
+  selector = `text=${transaction}`;
+  elementTx = await page.locator(selector).first();
+
+  selector = `text=${Wallets.richWalletAddress}`;
+  elementFrom = await page.locator(selector).first();
+
+  selector = `text=${Token.ERC20AddressETH}`;
+  elementTo = await page.locator(selector).first();
+
+  selector = `text=${Values.txSumETH}`;
+  element = await page.locator(selector).first();
+
+  //Check tx Hash
+  await expect(elementTx).toBeVisible(config.extraTimeout);
+  //Check address From
+  await expect(elementFrom).toBeVisible(config.extraTimeout);
+  //Check address To
+  await expect(elementTo).toBeVisible(config.extraTimeout);
+  //Check transaction Amount
+  await expect(element).toBeVisible(config.extraTimeout);
+});
+
+//@id1680
+test(" Check on BE Transfer custom ERC-20 token via Portal", async ({ page }) => {
+  transaction = await helper.readFile(Path.absolutePathToBufferFiles, Buffer.txMultiTransferCustomTokenI);
+  const adressTo = await helper.readFile(Path.absolutePathToBufferFiles, Buffer.L2);
+  url = BlockExplorer.baseUrl + `/tx/${transaction}` + BlockExplorer.localNetwork;
+
+  await page.goto(url);
+
+  selector = `text=${transaction}`;
+  elementTx = await page.locator(selector).first();
+
+  selector = `text=${Wallets.richWalletAddress}`;
+  elementFrom = await page.locator(selector).first();
+
+  selector = `text=${adressTo}`;
+  elementTo = await page.locator(selector).first();
+
+  selector = `text= 1 `;
```

Comment:
Is this a correct selector, do we need these extra spaces?

Reply:
Fixed - removed extra spaces.